lib/Doctrine/Annotations/AnnotationReader.php

```diff
@@ -86,7 +86,9 @@ class AnnotationReader implements Reader
         // Symfony 3.3 Cache Adapter
         'experimental' => true,
         // Slevomat Coding Standard
-        'phpcsSuppress' => true
+        'phpcsSuppress' => true,
+        // PHP CodeSniffer
+        'codingStandardsIgnoreStart' => true, 'codingStandardsIgnoreEnd' => true
```

Comment:
Please break the line here.

Reply:
@lcobucci , are you sure about this? 
Above my changes tags from one lib is grouped in same way ([example](https://github.com/doctrine/annotations/blob/320af0032a927aece85014c9594badb0df631556/lib/Doctrine/Annotations/AnnotationReader.php#L77))

Reply:
Not all of them, also it'll be changed once we make it follow our CS. :)

Reply:
Ok). Done